source/_integrations/inels.markdown

```diff
@@ -0,0 +1,72 @@
+---
+title: iNELS
+description: Instructions on how to integrate iNELS with Home Assistant.
+ha_category:
+  - Binary Sensor
+  - Button
+  - Climate
+  - Cover
+  - Light
+  - Select
+  - Sensor
+  - Switch
+ha_release: '2023.1'
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@nejezchleb'
+  - '@jhoralek'
+  - '@zed4805'
+ha_domain: inels
+ha_platforms:
+  - binary-sensor
+  - button
+  - climate
+  - cover
+  - light
+  - select
+  - sensor
+  - switch
```

Comment:
Same as above, a parent PR in Core can only contain a single platform, so this list seems not correct.